### ISSUE TITLE
CAS  as oidc client causes NoClassDefFoundError: net/minidev/json/JSONAware

### ISSUE THREAD

**Issue by aanli** [apereo/cas#1970]
I follow https://apereo.github.io/cas/4.2.x/integration/Delegate-Authentication.html
It works well with Oauth server (facebook) and with CAS server. None with OIDC (google)

- CAS server version : 4.2.4
- Container/environment type/version: test with tomcat 7.0.69 and tomcat 8.5.4 ; java  1.7.0_95 ant java 1.8.0_101 ; ubuntu 14.04
- Steps to duplicate the problem : <br/>
 a. include dependency cas-server-support-pac4j-webflow <br/>
 b. in cas.properties,
```
 cas.pac4j.oidc.id=<xxx>.apps.googleusercontent.com
 cas.pac4j.oidc.secret=<xxx>
 cas.pac4j.oidc.discoveryUri=https://accounts.google.com/.well-known/openid-configuration
```
   c. click on "Oidc"

- In tomcat, I have no error message
- In jetty (mvnw jetty:run-forked), I have some errors:
[STDERR] org.springframework.web.util.NestedServletException: Handler processing failed; nested exception is java.lang.NoClassDefFoundError: net/minidev/json/JSONAware
...
[STDERR] java.lang.ClassNotFoundException: net.minidev.json.JSONAware
...
 
Discussions in https://groups.google.com/a/apereo.org/forum/?utm_medium=email&utm_source=footer#!msg/cas-user/lOVuZpCLqLc/EfJoLoEFBAAJ

**Comment by mmoayyed** [comment]
Overlay https://github.com/cas-projects/cas-issues/pull/15

**Comment by mmoayyed** [comment]
Fixed in 4.2.5
